logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/CallRepository.kt

```diff
@@ -563,6 +563,10 @@ internal class CallDataSource(
             callJobs[conversationId] = scope.launch {
                 observeEpochInfo(conversationId).onSuccess {
                     it.collectLatest { epochInfo ->
+                        callingLogger.i(
```

Comment:
We can use `logStructuredJson`, so this can be searchable in datadog later for the internal builds.

Reply:
https://github.com/wireapp/kalium/blob/1070f5928925ba5cb4f44f57c02e6fde6385c761/logic/src/commonMain/kotlin/com/wire/kalium/logic/CoreLogger.kt#L61-L62